crates/ops-lambdas/indexer-monitor/src/handler.rs

```diff
@@ -0,0 +1,311 @@
+            .await
+            .context("Failed to fetch fulfilled requests number for client {client}")
+            .map_err(|e| Error::from(e.to_string()))?;
+        if let Err(e) = publish_metric(
+            &config.region,
+            &config.namespace,
+            format!("fulfilled_requests_number_from_{client}").as_str(),
+            fulfilled_count as f64,
+            now,
+        )
+        .await
+        {
+            error!(error = %e, "Failed to publish metric");
+        };
+    }
+
+    for prover in event.provers {
+        debug!(prover, "Processing prover");
+
+        let address = Address::from_str(&prover)
+            .context("Failed to parse prover address")
+            .map_err(|e| Error::from(e.to_string()))?;
+
+        let fulfilled_number = monitor
+            .fetch_fulfillments_number_by_prover(start_time, now, address)
+            .await
+            .context("Failed to fetch fulfilled requests number by prover {prover}")
+            .map_err(|e| Error::from(e.to_string()))?;
+        if let Err(e) = publish_metric(
+            &config.region,
+            &config.namespace,
+            format!("fulfilled_requests_number_by_{prover}").as_str(),
+            fulfilled_number as f64,
+            now,
+        )
+        .await
+        {
+            error!(error = %e, "Failed to publish metric");
+        };
+
+        let locked_number = monitor
+            .fetch_locked_number_by_prover(start_time, now, address)
+            .await
+            .context("Failed to fetch locked requests number by prover {prover}")
+            .map_err(|e| Error::from(e.to_string()))?;
+        if let Err(e) = publish_metric(
+            &config.region,
+            &config.namespace,
+            format!("locked_requests_number_by_{prover}").as_str(),
+            locked_number as f64,
+            now,
+        )
+        .await
+        {
+            error!(error = %e, "Failed to publish metric");
+        };
+
+        let slashed_number = monitor
+            .fetch_slashed_number_by_prover(start_time, now, address)
+            .await
+            .context("Failed to fetch slashed requests number by prover {prover}")
+            .map_err(|e| Error::from(e.to_string()))?;
+        if let Err(e) = publish_metric(
+            &config.region,
+            &config.namespace,
+            format!("slashed_requests_number_by_{prover}").as_str(),
+            slashed_number as f64,
+            now,
+        )
+        .await
+        {
+            error!(error = %e, "Failed to publish metric");
+        };
+    }
+
+    Ok(())
+}
+
+/// Publishes a metric to CloudWatch
+#[instrument(skip(region, namespace), err)]
+async fn publish_metric(
+    region: &str,
+    namespace: &str,
+    metric_name: &str,
+    value: f64,
+    timestamp: i64,
+) -> Result<(), Error> {
+    // Configure AWS SDK
+    let config = aws_config::from_env().region(Region::new(region.to_string())).load().await;
```

Comment:
We should configure retries + backoff for the AWS SDK. I think it defaults to some number, but in my experience its never enough, good to up it